grammars/asciidoc.cson

```diff
@@ -381,8 +381,9 @@
       #
       #   `Text in {backticks}` renders exactly as entered, in `monospace`.
       {
-        'match': '(?<=^|\\s)`[^`]+`'
-        'name': 'markup.raw.inline.asciidoc'
+        'match': '(^|[^\\p{Word};:"\'`\\}])((?:\\[([^\\\\]]+?)\\])?`(\\S|\\S.*?\\S)`(?![\\p{Word}"\'`]))'
```

Comment:
Now that we know we can use regexp groups like `\p{Word}`, we should probably start using them more consistently throughout the grammar. The benefit of these groups is that they are friendly to all languages (not just English).